src/slices/BondSlice.ts

```diff
@@ -151,12 +154,12 @@ export const calcBondDetails = createAsyncThunk(
     return {
       bond: bond.name,
       bondDiscount,
-      debtRatio,
-      bondQuote,
+      debtRatio: Number(debtRatio.toString()),
+      bondQuote: Number(bondQuote.toString()),
       purchased,
-      vestingTerm: Number(terms.vestingTerm),
-      maxBondPrice: maxBondPrice / Math.pow(10, 9),
-      bondPrice: bondPrice / Math.pow(10, 18),
+      vestingTerm: Number(terms.vestingTerm.toString()),
+      maxBondPrice: Number(maxBondPrice.toString()) / Math.pow(10, 9),
+      bondPrice: Number(bondPrice.toString()) / Math.pow(10, 18),
```

Comment:
I was concerned with converting maxBondPrice & bondPrice to a number, but in my testing it didn't seem to be a problem... feels like there may be some edge case where the precision loss is an issue, but maybe I'm wrong.